tests/sentry/integrations/github_enterprise/testutils.py

```diff
@@ -212,7 +212,7 @@
   }
 }"""
 
-PUSH_EVENT_EXAMPLE_INSTALLATION = b"""{
+PUSH_EVENT_EXAMPLE_INSTALLATION = r"""{
```

Comment:
@cclauss why this change? Actually this prompted me to take a brief look, and I've noticed a mixture of byte string and raw string literals in `tests/sentry/integrations`. @MeredithAnya do you know anything about why some are one kind and some are the other?

Reply:
interesting https://github.com/getsentry/sentry/pull/14924/files originally was r